.config/README.md

```diff
@@ -0,0 +1,164 @@
+# Default build configuration by Grafana
+
+**This is an auto-generated directory and is not intended to be changed! ⚠️**
+
+The `.config/` directory holds basic configuration for the different tools
```

Comment:
Well, it would display better on smaller screens but on big ones I would have to do a lot of horizontal reading. I think Grafana set a 120 char limit on the line length (this whole file was auto generated by the scaffolding tool https://grafana.com/developers/plugin-tools/#quick-start) if it's good enough for code readability I don't see why it shouldn't be good enough for documentation readability. Happy to change if it's an actual issue with your screen though